jdaviz/configs/specviz/plugins/unit_conversion/tests/test_unit_conversion.py

```diff
@@ -1,62 +1,30 @@
 import numpy as np
-from .. import unit_conversion as uc
-from astropy.nddata import (VarianceUncertainty, StdDevUncertainty,
-                            InverseVariance, UnknownUncertainty)
-
-
-SPECTRUM_SIZE = 10  # length of spectrum
+import pytest
+from astropy import units as u
+from astropy.nddata import UnknownUncertainty
+from astropy.tests.helper import assert_quantity_allclose
 
-unit_exponents = {StdDevUncertainty: 1,
-                  InverseVariance: -2,
-                  VarianceUncertainty: 2}
+from .. import unit_conversion as uc
```

Comment:
Usually local import is the last. It goes in 3 groups: stdlib, third-party, and then local